package.json

```diff
@@ -7,6 +7,7 @@
     "@material-ui/core": "^3.9.2",
     "@material-ui/icons": "^3.0.2",
     "date-fns": "^2.0.0-alpha.25",
+    "firebase": "^5.8.4",
```

Comment:
We should probably remove this dependency once we have it working in the backend repo.